fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/load/ChunkReader.java

```diff
@@ -49,14 +49,19 @@
 
     @Inject
     @Any
-    @BatchProperty(name = OperationFields.PARTITTION_WORKITEM)
+    @BatchProperty(name = OperationFields.PARTITION_WORKITEM)
     private String workItem;
 
     @Inject
     @Any
     @BatchProperty(name = OperationFields.PARTITION_RESOURCETYPE)
     private String resourceType;
 
+    @Inject
+    @Any
+    @BatchProperty(name = OperationFields.PARTITION_MATRIX)
+    private String matrix;
```

Comment:
is there a good explanation of what this is somewhere?

Reply:
Added comments to the OperationFields and README